.github/workflows/dapr.yml

```diff
@@ -71,6 +71,9 @@ jobs:
         uses: golangci/golangci-lint-action@v2.2.1
         with:
           version: ${{ env.GOLANGLINT_VER }}
+      - name: Run go mod tidy check diff
+        if: matrix.target_arch == 'amd64' && matrix.target_os == 'linux'
+        run: make modtidy check-diff
```

Comment:
If you want to output an error message that doesn't require looking at the log of failing workflow:

```
echo "::error file=optional.js,line=10,col=15::Something went wrong"
```
(all options are optional)
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-error-message